mne/gui/_file_traits.py

```diff
@@ -472,6 +492,12 @@ class SubjectSelectorPanel(HasPrivateTraits):
                             enabled_when='can_create_fsaverage'),
                        show_labels=False))
 
+    @on_trait_change('subjects_dir')
+    def _emit_subject(self):
+        # This silliness is the only way I could figure out to get the
+        # on_trait_change('subject_panel.subject') in CoregFrame to work!
+        self.subject = self.subject
+
```

Comment:
I'm not sure it will work. I tried about 100 things to get the title to always update and this finally worked. So let's unify separately in case it's even more work.